packages/react/src/Bento/Bento.features.stories.tsx

```diff
@@ -0,0 +1,171 @@
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={5} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+  </Bento>
+)
+
+export const Example: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 7}}
+      rowSpan={{xsmall: 3, small: 4}}
+      flow={{xsmall: 'row', small: 'row'}}
+      colorMode={ColorModesEnum.DARK}
+      visualAsBackground
+      className={styles['purple-background']}
+    >
+      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} fixedBottomLink leadingVisual={<CopilotIcon />}>
+        <Bento.Heading as="h4" size="5">
+          Unlocking innovation at scale with AI-driven software development.
+        </Bento.Heading>
+        <Link href="#">Learn more about Copilot</Link>
+      </Bento.Content>
+    </Bento.Item>
+    <Bento.Item columnSpan={{xsmall: 12, small: 5}} rowSpan={{xsmall: 3, small: 4}} colorMode={ColorModesEnum.DARK}>
+      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} horizontalAlign={'center'}>
+        <Bento.Heading as="h4" size="1">
+          88%
+        </Bento.Heading>
+        <Text align="center">of developers experience increased productivity.</Text>
+      </Bento.Content>
+      <Bento.Visual padding="normal" fillMedia={false}>
+        <img src={platformAI} alt="Platform Artificial Intelligence Logo" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={{xsmall: 3, small: 4}} visualAsBackground>
+      <Bento.Visual>
+        <img
+          src={analyzeAndDebug}
+          alt="A green and purple gradient screen with a input that has the placeholder, ask a question or type '/' for topics"
+        />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item
+      columnSpan={12}
+      rowSpan={{xsmall: 6, small: 4}}
+      flow={{xsmall: 'row', small: 'column'}}
+      colorMode={ColorModesEnum.DARK}
+    >
+      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} fixedBottomLink>
+        <Bento.Heading as="h4" size="5">
+          Mercado Libre frees developers minds to focus on their missions with GitHub.
+        </Bento.Heading>
+        <Link href="#">Read customer story</Link>
+      </Bento.Content>
+      <Bento.Visual padding="condensed">
+        <img src={mercardo} alt="A white man with grey hair and a beard, running a coffee stand, waving." />
+      </Bento.Visual>
+    </Bento.Item>
```

Comment:
```suggestion
    <Bento.Item
      columnSpan={{xsmall: 12, medium: 7}}
      rowSpan={{xsmall: 3, small: 5}}
      flow={{xsmall: 'row', small: 'row'}}
      colorMode={ColorModesEnum.DARK}
      visualAsBackground
      className={styles['purple-background']}
    >
      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} fixedBottomLink leadingVisual={<CopilotIcon />}>
        <Bento.Heading as="h4" size="5">
          Unlocking innovation at scale with AI-driven software development.
        </Bento.Heading>
        <Link href="#">Learn more about Copilot</Link>
      </Bento.Content>
    </Bento.Item>
    <Bento.Item columnSpan={{xsmall: 12, medium: 5}} rowSpan={{xsmall: 3, small: 5}} colorMode={ColorModesEnum.DARK}>
      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} horizontalAlign={'center'}>
        <Bento.Heading as="h4" size="1">
          88%
        </Bento.Heading>
        <Text align="center">of developers experience increased productivity.</Text>
      </Bento.Content>
      <Bento.Visual padding="normal" fillMedia={false}>
        <img src={platformAI} alt="Platform Artificial Intelligence Logo" />
      </Bento.Visual>
    </Bento.Item>
    <Bento.Item columnSpan={12} rowSpan={{xsmall: 3, small: 5}} visualAsBackground>
      <Bento.Visual>
        <img
          src={analyzeAndDebug}
          alt="A green and purple gradient screen with a input that has the placeholder, ask a question or type '/' for topics"
        />
      </Bento.Visual>
    </Bento.Item>
    <Bento.Item columnSpan={12} rowSpan={6} flow={{xsmall: 'row', medium: 'column'}} colorMode={ColorModesEnum.DARK}>
      <Bento.Content padding={{xsmall: 'normal', small: 'spacious'}} fixedBottomLink>
        <Bento.Heading as="h4" size="5">
          Mercado Libre frees developers minds to focus on their missions with GitHub.
        </Bento.Heading>
        <Link href="#">Read customer story</Link>
      </Bento.Content>
      <Bento.Visual padding="condensed">
        <img src={mercardo} alt="A white man with grey hair and a beard, running a coffee stand, waving." />
      </Bento.Visual>
    </Bento.Item>
```

Suggesting some updates to better align the examples with the designs (taller Bento items) and improving their responsive layout (stacking items under `medium` breakpoints)